sklearn/preprocessing/_data.py

```diff
@@ -291,6 +290,10 @@ class MinMaxScaler(OneToOneFeatureMixin, TransformerMixin, BaseEstimator):
     This transformation is often used as an alternative to zero mean,
     unit variance scaling.
 
+    MinMaxScaler doesn't reduce the effect of outliers; it only linearily
+    scales them down. For an example visualization, refer to :ref:`Compare
```

Comment:
This statement holds for all scalers (`StandardScaler`, `RobustScaler`, `MaxAbsScaler` and `MinMaxScaler`). What is different is that the scale value found by `RobustScaler` is not sensitive to the presence of a few large marginal outliers while it is for `StandardScaler` and even more so for `MinMaxScaler` and `MaxAbsScaler`.

Reply:
Yes, I see. To express how the `MinMaxScaler` differs from the other scalers concerning outliers, I have tried to come up with a now wording:

```
    `MinMaxScaler` doesn't reduce the effect of outliers, but it linearily
    scales them down into a fixed range, where the largest occuring data point
    corresponds to the maximum value and the smallest one corresponds to the
    minimum value.
```
    
What do you think?